LandSandBoat/server
# Mamool Ja Family Improvements

**_I affirm:_**

## What does this pull request do?

This includes several changes related to the Mamool Ja family.
- Makes it possible to break their weapons with critical hits
- Improves weapon throwing (previously they'd play the weapon break animation at the same time as the weapon throw skill animation)
- Fixed when some skills could be used based on weapon state
- Assigned correct mob skill list to mage Mamool Ja (blue ones)
- Changed many default animation subs from 4 to 0
- Adjusted mob skills according to Jimmayus' document:
https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit?gid=57955395#gid=57955395

## Steps to test these changes

Travel to any location with some Mamool Ja. Their weapons can be broken from critical hits (10% chance). Use `!tp 3000` repeatedly to test their weapons skills and see that they use them appropriately based on the weapon state.
